### PR TITLE
Shwe

### PR DESCRIPTION
**Pull request by shweaye810**
make settings more clear with numbers and operators.

### CODE REVIEW

File: app/settings.html [hunk 1]
```diff
@@ -23,13 +19,9 @@
       </p>
       <h2>breaks</h2>
       <p>
-        <span id="breakDurationMinus" class="pointer">-</span>
-        <span id="breakDuration"></span>
-        <span id="breakDurationPlus" class="pointer">+</span>
+        <input type="number" id="breakDuration" min="0" max="59" step="1" value=5>
```
Comment (hovancik) [inline app/settings.html]: Did you check how it looks? It's different after your changes. 

screen shot 2017-03-23 at 21 13 07

Also step should not be 1. Check how previous version was. 

That is why I am doing it how I am doing it. It looks better and I do not need to write css for inputs.

File: app/break.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
   </head>
   <body>
     <div class="microbreak">
+      <h2> Time to take a break! </h2>
```
Comment (hovancik) [inline app/break.html]: I think we do not need this, we can separate between break and microbreak very easy: 
- microbreak - one short sentence
- break - title and sentence

I prefer not to introduce this.

Comment (shweaye810) [inline app/break.html]: Oh, ok. I didn't know that. I think it should mention somewhere.

File: app/settings.html (hunk 1 above)
Comment (shweaye810) [inline app/settings.html]: Steps can be fixed very easily. While I was using the app, I was confused by the positive and negative sign, I wasn't even sure if I can click it.

Comment (shweaye810) [inline app/settings.html]: Actually, I want to work on showing the timer, but I am just trying out some other code and decided to change it a little.

Comment (shweaye810) [inline app/settings.html]: IMO, the text should be different with something clickable, or else it's confusing.

**Comment by hovancik** [comment]
Project has moved on a bit since this PR: there is time left in break, so I am closing this.

**Pull request closed without merging**